main.py

```diff
@@ -12,17 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Source code from https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/pubsub/main.py
+# The code in this module is based on
+# https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/pubsub/main.py.
+# See https://cloud.google.com/run/docs/tutorials/pubsub for the accompanying
+# Cloud Run/PubSub solutions guide.
+
+"""Runs Cloud Monitoring Notification Integration app with Flask."""
 
 # [START run_pubsub_server_setup]
-import base64
 import os
+import json
 
 from flask import Flask, request
-from config import configs
-
 from dotenv import load_dotenv
 
+import philips_hue
+from config import configs
+import pubsub
+
 
 load_dotenv()
```

Comment:
I included it because I need to be able to determine which config to load based on the environment, which is stored in .env as 'FLASK_APP_ENV' (I need to be able to run this line in main: `env = os.environ.get('FLASK_APP_ENV', 'default')`).  Would it be better to have get 'FLASK_APP_ENV' in config and then make that an option in the configs dictionary so main can just load `app.config.from_object(configs['FLASK_AP_ENV']`)?

Reply:
Done. I decided not to use frozendict since pytest gave me a warning about a deprecated import in the library, but I made the mapping dictionary private.